covasim/parameters.py

```diff
@@ -34,6 +34,10 @@ def make_pars(set_prognoses=False, prog_by_age=True, use_layers=False, **kwargs)
     pars['n_days']     = 60 # Number of days of run, if end_day isn't used
     pars['rand_seed']  = 1 # Random seed, if None, don't reset
     pars['verbose']    = 1 # Whether or not to display information during the run -- options are 0 (silent), 1 (default), 2 (everything)
+    pars['resample']   = 0 # Enable dynamic resampling
+    pars['resample_max_pop'] = 100000 # Maximum real population for resampling
+    pars['resample_threshold'] = 0.95 # Fraction susceptible population that will trigger resampling
+    pars['resample_factor'] = 2 # Factor by which we multiply population
```

Comment:
I think we should express this in terms of number infected instead of number susceptible, since they drive sim time. So I'd suggest just
```
    pars['resample_threshold'] = 0.1 # Fraction non-susceptible population that will trigger resampling
```
i.e., when you hit 10% non-susceptibles, you rescale the population. I don't think you need `pars['resample_max_pop']` , this is just going to be `pop_size`